HELP

```diff
@@ -55,3 +55,14 @@ Variables:
                    ** PASSWORD=Secret123 SAMPLE=ephemeral-storage make recreate-sample-idm
   CONFIG           The configuration to be used; by default 'default' is used, but
                    you can override setting it to 'default-hostpath'.
+
+                   ** ONLY FOR SINGLE NODE CLUSTERS AND FOR DEVELOPING **
+                   ** ACTUALLY IT NEEDS TO SET SELINUX IN PERMISSIVE MODE **
+                   **   oc debug nodes/....
+                   **   chroot /host
+                   **   setenforce Permissive
+                   **
+                   ** FINALLY: CONFIG=default-hostpath make deploy-cluster
```

Comment:
This change does not make sense; It could be that I messed up mixing another task here; Indeed what it is commented does not match with the manual settings explained for using hostpaths in PV for SNO scenario.

I will update it and I will delete it; sorry for this.